docs/source/getting_started/getting_started_hard_way.rst

```diff
@@ -0,0 +1,269 @@
+  build-essential automake libtool \
+  libssl-dev zlib1g-dev \
+  libc6-dbg golang \
+  git ssh tar gzip ca-certificates \
+  wget curl file unzip \
+  python
+
+.. note::  If you are willing to actively develop Iroha, please consider installing the `latest release <https://cmake.org/download/>`_ of CMake.
+
+macOS
+^^^^^
+If you are a happy macOS user and are willing to play with Iroha without building it, you can install our official Homebrew package and go straight to `Playing around with settings` section
+
+.. code-block:: shell
+
+  brew tap soramitsu/iroha
+  brew install iroha
+
+If you want to build it from scratch and actively develop it, please use this code to install all dependencies with Homebrew.
+
+.. code-block:: shell
+
+  xcode-select --install
+  brew install cmake boost postgres grpc autoconf automake libtool golang libpqxx
+
+.. hint:: To install the Homebrew itself please run 
+
+  ``ruby -e "$(curl -fsSL https://raw.githubusercontent.com/homebrew/install/master/install)"``
+
+Getting Iroha build
+-------------------
+
+Cloning the repository
+^^^^^^^^^^^^^^^^^^^^^^
+Clone the `Iroha repository <https://github.com/hyperledger/iroha>`_ to the directory of your choice.
+
+.. code-block:: shell
+
+  git clone -b develop https://github.com/hyperledger/iroha
+  cd iroha
+
+.. hint:: If you have installed the prerequisites with Docker, you don't need to clone Iroha again
+
+
+Building Iroha
+^^^^^^^^^^^^^^
+To build Iroha, use those commands
+
+.. code-block:: shell
+
+  cmake -H. -Bbuild;
+  cmake --build build -- -j$(nproc)
+
+.. note::  On macOS ``$(nproc)`` variable does not work. Check number of logical cores with ``sysctl -n hw.ncpu`` and put it explicitly in the command above, e.g. ``cmake --build build -- -j4``
+
+CMake parameters
+^^^^^^^^^^^^^^^^
+
+We use CMake to build platform-dependent build files. It has numerous of the flags for configuring the final build. Note that beside the listed params cmake's vars can be useful as well. Also as long as this page can be deprecated (or just not complete) you can browse custom flags via ``cmake -L``, ``cmake-gui``, or ``ccmake``.
+
+.. hint::  You can specify parameters at the cmake configuring stage (e.g cmake -DTESTING=ON).
+
+Main params
+"""""""""""
+
++--------------+-----------------+---------+------------------------------------------------------------------------+
+| Parameter    | Possible values | Default | Description                                                            |
++==============+=================+=========+========================================================================+
+| TESTING      |      ON/OFF     | ON      | Enables or disables build of the tests                                 |
++--------------+                 +---------+------------------------------------------------------------------------+
+| BENCHMARKING |                 | OFF     | Enables or disables build of the benchmarks                            |
```

Comment:
…which is what? 🤔 I think that we should tell that it disables build of benchmarking library, not benchmarks

Reply:
@l4l